src/pages/home/report/ReportActionsList.js

```diff
@@ -64,147 +70,132 @@ const defaultProps = {
     isLoadingMoreReportActions: false,
 };
 
-class ReportActionsList extends React.Component {
-    constructor(props) {
-        super(props);
-        this.renderItem = this.renderItem.bind(this);
-        this.keyExtractor = this.keyExtractor.bind(this);
-
-        this.state = {
-            fadeInAnimation: new Animated.Value(0),
-            skeletonViewHeight: 0,
-        };
-    }
-
-    componentDidMount() {
-        this.fadeIn();
-    }
-
-    fadeIn() {
-        Animated.timing(this.state.fadeInAnimation, {
-            toValue: 1,
-            duration: 100,
-            useNativeDriver: true,
-        }).start();
-    }
+/**
+ * Create a unique key for each action in the FlatList.
+ * We use the reportActionID that is a string representation of a random 64-bit int, which should be
+ * random enough to avoid collisions
+ * @param {Object} item
+ * @param {Object} item.action
+ * @return {String}
+ */
+function keyExtractor(item) {
+    return item.reportActionID;
+}
```

Comment:
Why this function is defined outside `ReportActionsList` Wouldn't it be better to define it inside and `useCallback` with empty dependency array.

Reply:
Just thought that since `keyExtractor` was a part of the class component it should still be a part of the functional component but maybe having that function a part of the class component was a "mistake" in the first place.

I think having the function defined outside is slightly better e.g. If we render 100 instances of that component all will use the same function, where if we defined it inside each will use it's own function (100 functions that are all the same).